CMakeLists.txt

```diff
@@ -25,10 +25,8 @@ add_subdirectory(atmos_cubed_sphere)
 if(INLINE_POST)
   set(BUILD_POSTEXEC OFF)
   add_subdirectory(upp)
-  set(POST_SRC io/inline_post.F90 io/post_nems_routines.F90 io/post_fv3.F90)
-else()
```

Comment:
Does this mean inline_post will be always on?

Reply:
@DusanJovic-NOAA May I know where INLINE_POST is defined?

Reply:
The default value is set here:
https://github.com/ufs-community/ufs-weather-model/blob/6ba0369075b40c084f0f18c8f7978c934329e60d/CMakeLists.txt#L64

and it is overridden in platform specific files, for example here:
https://github.com/ufs-community/ufs-weather-model/blob/6ba0369075b40c084f0f18c8f7978c934329e60d/cmake/configure_hera.intel.cmake#L1

Reply:
@DusanJovic-NOAA Got it. Thanks!